transmartApp/grails-app/init/org/transmartproject/app/BootStrap.groovy

```diff
@@ -50,11 +50,6 @@ class BootStrap {
 
         // force marshaller registrar initialization
         grailsApplication.mainContext.getBean 'marshallerRegistrarService'
-
-        if ('clientCredentialsAuthenticationProvider' in
-                grailsApplication.config.grails.plugin.springsecurity.providerNames) {
-            OAuth2SyncService.syncOAuth2Clients()
-        }
     }
 
```

Comment:
Wait, the configFine check is already there in tm-server, duplicated. Todo: deduplicate it.